test/SlowTests/Client/TimeSeries/Session/TimeSeriesTypedSessionTests.cs

```diff
@@ -52,6 +53,29 @@ public class HeartRateMeasure : TimeSeriesEntry
             [TimeSeriesValue(0)] public double HeartRate;
         }
 
+        [Fact]
+        public async Task CanRegisterTimeSeries()
+        {
+            using (var store = GetDocumentStore())
+            {
+                await store.TimeSeries.Register<User, StockPrice>();
+                await store.TimeSeries.Register("Users", nameof(HeartRateMeasure), new[] {nameof(HeartRateMeasure.HeartRate)});
```

Comment:
It would make sense to also update the policies from here, no?

Reply:
`store.TimeSeries.AddPolicy<User,StockPrice>(TimeValue retention, TimeValue aggregation)`
`store.TimeSeries.AddRawPolicy<User,StockPrice>(TimeValue retention)`

do we want also update/remove?